utils/fixed_decimal/src/decimal.rs

```diff
@@ -1670,55 +1709,31 @@ fn test_signum_zero() {
 }
 
 #[test]
-fn test_pad_or_truncate_left() {
-    let mut dec = FixedDecimal::from(1000);
-    assert_eq!("1000", dec.to_string());
-
-    dec.pad_or_truncate_left(-1);
-    assert_eq!("000", dec.to_string());
-
-    dec.pad_or_truncate_left(-10000);
-    assert_eq!("0", dec.to_string());
-
-    dec.pad_or_truncate_left(3);
-    assert_eq!("0000", dec.to_string());
-
+fn test_pad() {
     let mut dec = FixedDecimal::from_str("-0.42").unwrap();
     assert_eq!("-0.42", dec.to_string());
 
-    dec.pad_or_truncate_left(-1);
+    dec.padded_left(0);
     assert_eq!("-0.42", dec.to_string());
 
-    dec.pad_or_truncate_left(3);
+    dec.padded_left(3);
     assert_eq!("-0000.42", dec.to_string());
 
-    dec.pad_or_truncate_left(-3);
-    assert_eq!("-0.42", dec.to_string());
+    dec.padded_left(1);
+    assert_eq!("-00.42", dec.to_string());
 }
 
 #[test]
-fn test_padded_or_truncated_left() {
-    let dec = FixedDecimal::from(1000);
+fn test_truncate() {
```

Comment:
Nit: Please add some more tests for behavior of numbers whose most significant digit has a magnitude of -1 or lower.